packages/design-system/src/components/Table/Table.test.tsx

```diff
@@ -1,149 +1,70 @@
-import { mount, shallow } from 'enzyme';
 import React from 'react';
+import { render, screen } from '@testing-library/react';
+
 import Table from './Table';
 import TableCaption from './TableCaption';
+// import TableBody from './TableBody';
 
-const defaultCaptionChildren = 'Foo';
-const tableCaption = <TableCaption>{defaultCaptionChildren}</TableCaption>;
-
-function render(customProps = {}, children = tableCaption, deep = false) {
+const makeTable = (customProps = {}) => {
```

Comment:
![ron-approves](https://user-images.githubusercontent.com/7595652/186992475-f7db3898-f49e-4c58-a682-1c159b1afead.png)

Reply:
That is, he approves of _make table_

Reply:
Hahaha nice